modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -611,6 +627,38 @@ The cache performs a recursive directory inspection during the cache trim. The i
 
 ---
 
+=== cloud_storage_cache_trim_threshold_percent_objects
+
+Trim is triggered when the cache reaches this percent of the maximum object count. If this is unset, the default behavior is to start trim when the cache is about 100% full.
```

Comment:
@Eng/SME description is the same as cloud_storage_cache_trim_threshold_percent_size. Is this correct?

Reply:
No, one is object count and the other is size (bytes).

I think we should remove the word "about" here. It's not quite true that trim is triggered at 100%: we count reserved space as well as allocated space when doing the calculation. But I think saying "about" might leave users wondering what that really means.

Reply:
Updated the two properties below with: 

> Cache trimming is triggered when the number of objects in the cache reaches this percentage relative to it's maximum object count. If unset, the default behavior is to start trimming when the cache is full.


> Cache trimming is triggered when the cache size reaches this percentage relative to it's maximum capacity. If unset, the default behavior is to start trimming when the cache is full.